File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-0.py
class Solution:
    def replaceElements(self, arr: List[int]) -> List[int]:
        # 마지막 바로 전 인덱스까지 순회
        for i in range(len(arr)-1):
            arr[i] = max(arr[i+1:])
        
        # 마지막 원소는 -1로
        arr[-1] = -1
        
        return arr

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int tmp = 0, ans = 0;

        for(int i = 0; i < nums.size(); i++) {
            /* nums[1]가 1이면 tmp를 1만큼 증가 */
            if(nums[i] == 1)
                tmp++;
            /* 아니라면 ans와 값을 비교하고, tmp를 0으로 초기화 */
            else {
                ans = max(tmp, ans);
                tmp = 0;
            }
        }
        /* */
        return max(tmp, ans);
    }
};